nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml

```diff
@@ -440,6 +440,7 @@
                                                 <include>${staging.dir}/js/nf/canvas/nf-clipboard.js</include>
                                                 <include>${staging.dir}/js/nf/canvas/nf-custom-ui.js</include>
                                                 <include>${staging.dir}/js/nf/canvas/nf-canvas-utils.js</include>
+                                                <include>${staging.dir}/js/nf/canvas/nf-component-version.js</include>
```

Comment:
Got it. I feel like it wasn't possible when I originally typed it up. However, this module went through a couple iterations during the development of this feature. I'll try updating the dependency injection with your suggestions now.